src/modules/Makefile.am

```diff
@@ -170,6 +170,8 @@ job_info_la_SOURCES = \
 	job-info/watch.c \
```

Comment:
commit mesage for `job-info: support new update-lookup/watch service `: capitalize "it" in the 4th paragraph of the body.